### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by soniaturcotte**
@jonheslop CSS grid isn't working properly because of the way the files are set up (layout/events). I’m not sure whether to try and sort it out or just loose grid and use tachyons. Wyt?

### CODE REVIEW

File: views/layout.njk [hunk 1]
```diff
@@ -4,17 +4,24 @@
...
-  <body class="pa4">
-    <header class="mb5">
-      <h1 class="ma0 f-headline lh-solid">some gallery thing</h1>
-    </header>
-    <main>
-      {% block content %}{% endblock %}
-    </main>
+  
+  <body>
+    <div class="wrapper">
```
Comment (jonheslop) [inline views/layout.njk]: This is on the layout for the whole site

File: views/events/list.njk [hunk 2]
```diff
@@ -2,26 +2,31 @@
 
 {% block content %}
 
-  <article>
-    <h2 class="f3 fw4 pa3 mv0">Events</h2>
-    <div class="cf pa2">
-      <div class="fl w-50 w-25-m w-20-l pa2">
+  <article class="bl br bw2 b--white">
```
Comment (jonheslop) [inline views/events/list.njk]: putting wrapper here would apply the grid to each event

**Comment by soniaturcotte** [comment]
@jonheslop @tlwr  I’ve removed css grid and just used tachyons. I'm a bit stuck on the column height (the standard tachyon measure is either too small or too big) and the where to set the breakpoints on the columns.

Before I continue hacking into it, could you see if I have it right?

**Comment by tlwr** [comment]
image

This is what it looks like on my machine. The text overlaps in 2 different ways, one over the image and one over the dates. It that the column height problem you're looking at?

**Pull request closed without merging**